static/app/views/issueDetails/traceTimeline/traceIssue.tsx

```diff
@@ -40,29 +40,31 @@ export function TraceIssueEvent({event}: TraceIssueEventProps) {
           });
         }}
       >
-        <TraceIssueProjectBadge>
-          {project ? (
-            <ProjectBadge
-              project={project}
-              avatarSize={avatarSize}
-              hideName
-              disableLink
-            />
-          ) : (
-            <Placeholder
-              shape="rect"
-              width={`${projectBadgeSize}px`}
-              height={`${projectBadgeSize}px`}
-            />
-          )}
-        </TraceIssueProjectBadge>
-        <TraceIssueDetailsContainer>
-          <NoOverflowDiv>
-            <TraceIssueEventTitle>{title}</TraceIssueEventTitle>
-            <TraceIssueEventTransaction>{event.transaction}</TraceIssueEventTransaction>
-          </NoOverflowDiv>
-          <NoOverflowDiv>{subtitle}</NoOverflowDiv>
-        </TraceIssueDetailsContainer>
+        <RootContainer>
```

Comment:
Removing whitespaces makes reviewing this easier.
<img width="402" alt="image" src="https://github.com/getsentry/sentry/assets/44410/bf4aa3fa-1910-49e7-bbe5-263f0ce88e20">